features/guidance/collapse.feature

```diff
@@ -725,10 +725,10 @@ Feature: Collapse
 
         When I route I should get
             | waypoints | route                 | turns                                      |
-            | a,g       | road,cross,cross      | depart,turn left,arrive                    |
-            | a,e       | road,road,road        | depart,continue straight,arrive            |
+            | a,g       | road,cross,cross      | depart,fork left,arrive                    |
+            | a,e       | road,road,road        | depart,fork slight right,arrive            |
```

Comment:
The scenario in question still provides a now different test. Can you create a copy of this test with similar setup that again identifies a sliproad here? The feature in collapse is supposed to collapse, which is not happening now.
The test as it is here can be moved to a different feature set.